README.md

```diff
@@ -4,9 +4,16 @@ This module extends the comments module core functionality and adds the ability
 
 When a comment has been flagged, it cannot be flagged by anyone else.
 
+## Requirements
+
+- silverstripe/comments ^3.1
+- silverstripe/framework ^4.0
+
+Note: This branch is compatible with SilverStripe 4. For a SilverStripe 3 release, please see the 1.x release line.
+
 ## Installation
 
-`composer require micmania1/silverstripe-flagcomments`
+`composer require nzta/silverstripe-flagcomments`
```

Comment:
you need to create the repo under the NZTA first and migrate the current repo. otherwise composer not able to find the `nzta/silverstripe-flagcomments` repo. im not able to test this.

Reply:
Once you're happy with the PR, Michael will merge this, create separate tags, then switch ownership to NZTA